### ISSUE TITLE
Training loss is inf but validation loss is decreasing Deep Speech

### ISSUE THREAD

**Issue by alamnasim** [mozilla/DeepSpeech#1910]
Hi
we are trying to train the deep speech model for the Indian spoken language of ‘hinglish’. We have around 1300+ hours of data with 27 characters in the alphabet.txt and are using the following parameters :

./DeepSpeech.py
–train_files data/train.csv
–dev_files data/dev.csv
–test_files data/test.csv
–alphabet_config_path data/alphabet.txt
–checkpoint_dir ~/checkpoint_dir
–export_dir ~/saved_models
–lm_binary_path data/lm.binary
–lm_trie_path data/trie
–report_count 200
–show_progressbar true
**–train_batch_size 20
–test_batch_size 48
–dev_batch_size 48
–validation_step 1
–learning_rate 0.0001
–n_hidden 2048
–epoch 10
–dropout_rate 0.15**

The train and the validation losses are as follows:

Training of Epoch 0 - loss: inf
Validation of Epoch 0 - loss: 95.800559
Training of Epoch 1 - loss: inf
Validation of Epoch 1 - loss: 81.028593
Training of Epoch 2 - loss: inf
Validation of Epoch 2 - loss: 77.50257
Training of Epoch 3 - loss: inf
Validation of Epoch 3 - loss: 75.459322
Training of Epoch 4 - loss: inf
Validation of Epoch 4 - loss: 74.528508
…

previously We also fine tuned some parameters but got same training loss as inf in all the epochs:
–train_batch_size 32
–test_batch_size 64
–validation_step 1
–learning_rate 0.0001
–n_hidden 1024
–epoch 10
–dropout_rate 0.13

what changes can we do in the parameters to make this better?
Our dataset is not very clean, but not that bad either.

Any help is appreciated.

**Comment by kdavis-mozilla** [comment]
As this is not a but or a feature request, could you take the discussion to our [discourse forum](https://discourse.mozilla.org/c/deep-speech)? Thanks.
